### PR TITLE
Diagnose PubIntro

### PR DESCRIPTION
**Pull request by notlesh**
*poke*

### CODE REVIEW

File: llarp/dht/context.cpp [hunk 1]
```diff
@@ -532,7 +532,7 @@ namespace llarp
                                  const Key_t& tellpeer, bool relayed,
                                  uint64_t relayOrder)
     {
-      TXOwner asker(from, txid);
+      TXOwner asker(OurKey(), txid);
```
Comment (notlesh) [inline llarp/dht/context.cpp]: Note to self: consider this carefully when cherry-picking these commits

File: llarp/dht/messages/pubintro.hpp [hunk 2]
```diff
@@ -18,8 +18,7 @@ namespace llarp
       uint64_t relayOrder = 0;
       uint64_t txID       = 0;
       PublishIntroMessage(const Key_t& from, bool relayed_)
-          : IMessage(from)
-          , relayed(relayed_)
+          : IMessage(from), relayed(relayed_)
```
Comment (notlesh) [inline llarp/dht/messages/pubintro.hpp]: <3 make format

**Comment by notlesh** [comment]
image

**Comment by jagerman** [comment]
This was superceded by #1147 which (I believe) pulled all the useful bits out of here.  Closing.

**Pull request closed without merging**